lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -129,14 +126,8 @@ def _connect_drag_callback(self):
         def set_xlim(drag_event):
             # Callback for dragging the field of view
             old_xlims = np.array(self._axes.get_xlim())
-            self._dx = self._dx + drag_event.dx
-
-            # We don't need to update more than 30 times per second (1/30 = 0.033).
-            if abs(time.time() - self._last_update) > 0.033:
-                self._axes.set_xlim(old_xlims - self._dx)
-                self._dx = 0
-                self._last_update = time.time()
-                canvas.draw_idle()
+            self._axes.set_xlim(old_xlims - drag_event.dx)
+            canvas.draw_idle()
```

Comment:
Looks cleaner! Is the rate limitation for drawing the canvas now handled in the backend?